src/Raven.Server/Rachis/RachisLogHistory.cs

```diff
@@ -489,5 +489,32 @@ public unsafe bool HasHistoryLog(ClusterOperationContext context, BlittableJsonR
                 return true;
             }
         }
+
+
+        public unsafe bool TryGetCommandByIndex(ClusterOperationContext context, long index, out DynamicJsonValue cmd)
+        {
+            cmd = null;
+            var reversedIndex = Bits.SwapBytes(index);
+            using (Slice.External(context.Allocator, (byte*)&reversedIndex, sizeof(long), out var indexSlice))
+            {
+                var table = context.Transaction.InnerTransaction.OpenTable(LogHistoryTable, LogHistorySlice);
+
+                foreach (var result in table.SeekForwardFrom(LogHistoryTable.Indexes[LogHistoryIndexSlice], indexSlice, 0))
+                {
+                    var entryHolder = result.Result;
+                    var curIndex = ReadIndex(entryHolder);
```

Comment:
I don't see usages of `curIndex`, `guid`, `type` & `state`

Reply:
Done